UPGRADING.md

```diff
@@ -210,85 +160,74 @@ Several existing methods have been renamed, a full list of which is shown below:
 
 See the [full documentation](https://docs.bugsnag.com/platforms/android) for more information.
 
-## BeforeNotify/BeforeSend/Callback
+## Event callbacks
 
-These three callbacks have been superseded and replaced by `OnError`, a single callback which runs immediately after an error has occurred. This can run globally on all errors, or on an individual error.
+`BeforeNotify` and `BeforeSend` callbacks have been superseded and replaced by `OnError`, a single callback which runs immediately after an error has occurred. This can run globally on all errors, or on an individual handled error:
```

Comment:
IMO 'individual error' is clearer, particularly as we're about to allow end-users to alter the handled state of errors.